public/js/chat.js

```diff
@@ -35,6 +35,7 @@ back_btn.addEventListener('click', (event) => {
 });
 
 window.addEventListener('resize', () => {
+
   if (window.innerWidth > 750) {
     last_messages.setAttribute('style', 'display:block');
```

Comment:
this code is redundant you could use a function for these three lines and pass the value of display for everyone of them